src/handlebars/releases.handlebars

```diff
@@ -37,23 +37,48 @@
       <a id="description_link" href=""></a>
       <h3><a href="./release_notes.html">All Release Notes</a></h3>
 
+      <div class="filter-divs">
+        <div class="filter-os filter-div">
+          <span><strong>Operating System:</strong></span>
+          <select id="os-filter" class="filter" onchange="filterOS()">
+            <option value="Any">-Any-</option>
+            <option value="Linux">Linux</option>
+            <option value="macOS">macOS</option>
+            <option value="AIX">AIX</option>
+            <option value="Windows">Windows</option>
+            <option value="Solaris">Solaris</option>
+          </select>
+        </div>
+
+        <div class="filter-arch filter-div">
+          <span><strong>Architecture:</strong></span>
+          <select id="arch-filter" class="filter" onchange="filterArch()">
+            <option value="Any">-Any-</option>
+            <option value="x64">x64</option>
+            <option value="x32">x32</option>
+            <option value="s390x">s390x</option>
```

Comment:
Same here. This gets you into a state where when "Linux" is selected the architecture list shows this static list which is not very intuitive. At least it confused me. I'd have expected for it to refresh and only show a list of arches for the corresponding OS selection.